.github/workflows/docs-release.yml

```diff
@@ -49,13 +48,22 @@ jobs:
         name: Copy yaml
         run: |
           cp /tmp/buildx-docs/out/reference/*.yaml ./data/buildx/
+      -
+        name: Make vendoring temp dir
+        run: |
+          mkdir -p /tmp/hugo_vendor_output
```

Comment:
Maybe we could have a `make vendor` target upstream so we don't need to explicitly do this here and we keep it consistent?